templates/octaviaapi/config/httpd.conf

```diff
@@ -35,6 +35,8 @@ CustomLog /dev/stdout proxy env=forwarded
     CustomLog /dev/stdout combined env=!forwarded
     CustomLog /dev/stdout proxy env=forwarded
 
+    TimeOut {{ $.TimeOut }}
```

Comment:
2. there it's added to the template, without checking that the value is defined
without specifying the timeout in the spec, "TimeOut" is an empty string which is rejected by apache.

We can check that it's empty:

 oc get secret -n octavia-kuttl-tests  octavia-api-config-data -o json | jq -r '.data["httpd.conf"]' | base64 -d | grep TimeOut
    TimeOut 
    TimeOut